hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
+output = document.getElementById("result")
+
+function handleInput() {
+
+	if (elem.value < 0) {
+		output.style.color = "red"
+		output.innerHTML = "Please Enter a possitive number!"
+	}
+
+	else {
+		let elem_str = elem.value.toString()
+		let reverse_elem = elem_str.split('').reverse().join('')
+		console.log(reverse_elem)
+		if (elem_str == reverse_elem) {
+			output.style.color = "green"
+			output.innerHTML = "Yes. This is Pallindrome!";
+		}
+		else {
+			output.style.color = "red"
```

Comment:
When I run WAVE on bad input it shows a contrast error with this default value of 'red'. I did the same thing in my homework and a reviewer suggested using the bootstrap colors instead which will fix the contrast error. You can use the bootstrap colors by conditionally adding the bootstrap color classes `text-success` for green and `text-danger` for red. [Bootstrap reference](https://getbootstrap.com/docs/4.0/utilities/colors/)

Reply:
I have changed it but it is still throwing contrast error